### DISCUSSION TITLE
Themes

### DISCUSSION THREAD

**Discussion by praveenkumar277** [Yisus7u7/termux-desktop-lxqt#13] [Q&A]
How can i change the themes ?

**Comment by JesusChapman** [comment]
https://www.google.com/amp/s/www.addictivetips.com/ubuntu-linux-tips/customize-the-lxqt-desktop/amp/

**Comment by JesusChapman** [comment]
You can also download themes for Kvantum here: 

https://store.kde.org/browse?cat=123&order=latest
